README.md

```diff
@@ -46,6 +46,10 @@ You may find the secrets page by navigating to `Settings > Secrets > Actions` on
 
 **Optional** The paths to the files to sign.
 
+### `description`
+
+**Optional** The description of the signed content (same as /d for signtool).
```

Comment:
We should probably make it more clear that this is only used when something is signed by signtool?